frontend/apps/marketing/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@code-dot-org/marketing",
+  "version": "0.1.0",
+  "private": true,
+  "keywords": [
+    "code.org"
+  ],
+  "homepage": "https://github.com/code-dot-org/code-dot-org/blob/staging/frontend/apps/marketing/#readme",
+  "bugs": {
+    "url": "https://github.com/code-dot-org/code-dot-org/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/code-dot-org/code-dot-org.git",
+    "directory": "frontend/apps/marketing"
+  },
+  "license": "SEE LICENSE IN LICENSE",
+  "scripts": {
+    "build": "next build",
+    "dev": "next dev --turbopack",
+    "lint": "eslint .",
+    "lint:fix": "eslint --fix .",
+    "prettier": "prettier --check .",
+    "prettier:fix": "prettier --write .",
+    "start": "next start"
```

Comment:
Would it be possible to disambiguate this from `yarn start` in `apps` (looks like this requires an existing production build)?

Reply:
What do you think of using `yarn start` in frontend as well? I wonder if we should be consistent in the command names

Reply:
Yeah consistency sounds great to me :) -- that said, it does seem like what we call "yarn start" in apps is probably better described as "yarn dev" (it really calls `yarn grunt dev`), so aligning to that standard feels maybe backwards (ie, wouldn't the "right" thing be to rename `yarn start` to `yarn dev` in apps?) Anyway, buncha sharp developers work here, they'll figure it out right?